modules/nw-anp-banp-metrics.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * list of assemblies where this module is included:
+// networking/network_security/AdminNetworkPolicy/nw-ovn-k-anp-banp-metric.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="anp-banp-metrics_{context}"]
+= Metrics for AdminNetworkPolicy
```

Comment:
just say Metrics

you say ANP and BANP everywhere :P even if its not needed but then added ANP alone here while table has BANP as well

Perhaps just make it "Metrics" since your heading is already monitoring ...?

Reply:
Metrics is fine, but I've been eliminating ANP and BANP :P.